builtin/providers/nsone/resource_apikey.go

```diff
@@ -118,64 +120,64 @@ func resourceDataToPermissions(d *schema.ResourceData) nsone.PermissionsMap {
 	return p
 }
 
-func resourceDataToApikey(u *nsone.Apikey, d *schema.ResourceData) error {
-	u.Id = d.Id()
+func resourceDataToApikey(u *account.APIKey, d *schema.ResourceData) error {
+	u.ID = d.Id()
 	u.Name = d.Get("name").(string)
 	if v, ok := d.GetOk("teams"); ok {
 		teamsRaw := v.([]interface{})
-		u.Teams = make([]string, len(teamsRaw))
+		u.TeamIDs = make([]string, len(teamsRaw))
 		for i, team := range teamsRaw {
-			u.Teams[i] = team.(string)
+			u.TeamIDs[i] = team.(string)
 		}
 	} else {
-		u.Teams = make([]string, 0)
+		u.TeamIDs = make([]string, 0)
 	}
 	u.Permissions = resourceDataToPermissions(d)
 	return nil
 }
 
 // ApikeyCreate creates ns1 API key
 func ApikeyCreate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*nsone.APIClient)
-	mj := nsone.Apikey{}
+	client := meta.(*nsone.Client)
+	mj := account.APIKey{}
```

Comment:
i think `mj` is short for monitoring job. not a big deal at all, but maybe this should be `a` or `ak` for apikey. but again, nbd